docs/API.md

```diff
@@ -18,12 +18,16 @@ There are a number of methods that a DB storage backend should implement:
     * .createSessionToken(tokenId, sessionToken)
     * .updateSessionToken(tokenId, sessionToken)
     * .sessionToken(id)
+    * .sessionTokenVerified(tokenId)
```

Comment:
I'm assuming the two distinct methods are used to prevent hitting two DB tables unless strictly necessary. I also just found the `sessionTokenVerified` strategy.

`Verified` seems like an odd choice for a name, but nothing comes to mind that is more clear.